proto/vschema.proto

```diff
@@ -64,7 +64,11 @@ message Table {
   // shard, as dictated by the keyspace id.
   // The keyspace id is represened in hex form
   // like in keyranges.
-  string pinned =5;
+  string pinned = 5;
+  // is_authoritative is set ot true if columns is
```

Comment:
It seems to me that `is_authoritative` is too broad of an attribute name for this feature.
After all the vschema is always authoritative for the sharding controls, etc.

How about we keep this more focused and call the attribute "authoritative_columns" or something of that sort.

Also typo "ot" above.